go.mod

```diff
@@ -1,111 +1,98 @@
 module sigs.k8s.io/vsphere-csi-driver
```

Comment:
I made a PR to fix `v2`: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/156